components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/IDTokenResponseValidator.java

```diff
@@ -58,4 +63,23 @@ public void validateMethod(HttpServletRequest request) throws OAuthProblemExcept
     public void validateContentType(HttpServletRequest request) throws OAuthProblemException {
     }
 
-}
\ No newline at end of file
+    /**
+     * Method to check whether the scope parameter string contains 'openid' as a scope.
+     *
+     * @param scope
+     * @return
+     */
+    private static boolean containOIDCScope(String scope) {
```

Comment:
We don't have a Util package in oauth.common bundle, hence I added this private method since it will only be used by this class.
 
Yes, the idea behind adding this util method in this was to avoid taking the dependency from oauth bundle.